sympy/functions/elementary/tests/test_integers.py

```diff
@@ -395,9 +397,18 @@ def test_ceiling():
     assert (ceiling(y) < n) == (y <= n - 1)
     assert (ceiling(y) > n) == (y > n)
 
-    assert ceiling(RootOf(x**5-x**2+1,0)) == 0
+    assert ceiling(RootOf(x**3 - 27*x, 2)) == 6
     s = ImageSet(Lambda(n, n + (CRootOf(x**5 - x**2 + 1, 0))), Integers)
-    assert s.intersect(Interval(-10, 10)) == {-9 + CRootOf(x**5 - x**2 + 1, 0), -8 + CRootOf(x**5 - x**2 + 1, 0), -7 + CRootOf(x**5 - x**2 + 1, 0), -6 + CRootOf(x**5 - x**2 + 1, 0), -5 + CRootOf(x**5 - x**2 + 1, 0), -4 + CRootOf(x**5 - x**2 + 1, 0), -3 + CRootOf(x**5 - x**2 + 1, 0), -2 + CRootOf(x**5 - x**2 + 1, 0), -1 + CRootOf(x**5 - x**2 + 1, 0), CRootOf(x**5 - x**2 + 1, 0) + 1, CRootOf(x**5 - x**2 + 1, 0) + 2, CRootOf(x**5 - x**2 + 1, 0) + 3, CRootOf(x**5 - x**2 + 1, 0) + 4, CRootOf(x**5 - x**2 + 1, 0) + 5, CRootOf(x**5 - x**2 + 1, 0) + 6, CRootOf(x**5 - x**2 + 1, 0) + 7, CRootOf(x**5 - x**2 + 1, 0) + 8, CRootOf(x**5 - x**2 + 1, 0) + 9, CRootOf(x**5 - x**2 + 1, 0) + 10, CRootOf(x**5 - x**2 + 1, 0)}
+    assert s.intersect(Interval(-10, 10)) == {-9 + CRootOf(x**5 - x**2 + 1, 0),
+    -8 + CRootOf(x**5 - x**2 + 1, 0), -7 + CRootOf(x**5 - x**2 + 1, 0), -6 +
+    CRootOf(x**5 - x**2 + 1, 0), -5 + CRootOf(x**5 - x**2 + 1, 0), -4 +
+    CRootOf(x**5 - x**2 + 1, 0), -3 + CRootOf(x**5 - x**2 + 1, 0), -2 +
+    CRootOf(x**5 - x**2 + 1, 0), -1 + CRootOf(x**5 - x**2 + 1, 0), CRootOf(x**5 - x**2 + 1, 0) + 1,
+    CRootOf(x**5 - x**2 + 1, 0) + 2, CRootOf(x**5 - x**2 + 1, 0) + 3,
+    CRootOf(x**5 - x**2 + 1, 0) + 4, CRootOf(x**5 - x**2 + 1, 0) + 5,
+    CRootOf(x**5 - x**2 + 1, 0) + 6, CRootOf(x**5 - x**2 + 1, 0) + 7,
+    CRootOf(x**5 - x**2 + 1, 0) + 8, CRootOf(x**5 - x**2 + 1, 0) + 9,
+    CRootOf(x**5 - x**2 + 1, 0) + 10, CRootOf(x**5 - x**2 + 1, 0)}
```

Comment:
Take some time to consider how to format your code for readability. It is not hard to format this in a nicer way e.g.:
```
assert s.intersect(Interval(-10, 10)) == {
    -9 + CRootOf(x**5 - x**2 + 1, 0),
    -8 + CRootOf(x**5 - x**2 + 1, 0),
    ...
}
```
I would just use a comprehension though:
```
r = CRootOf(x**5 - x**2 + 1, 0)
assert s.intersect(Interval(-10, 10)) == {i + r for i in range(-9, 11)}
```